cost-analyzer/templates/etl-utils-deployment.yaml

```
@@ -47,6 +47,8 @@ spec:
           image: {{ .Values.kubecostModel.fullImageName }}
           {{- else if .Values.imageVersion }}
           image: {{ .Values.kubecostModel.image }}:{{ .Values.imageVersion }}
+        {{- else if eq "development" .Chart.AppVersion }}
+        - image: gcr.io/kubecost1/cost-model-nightly:latest
```

I think this might be in error
```suggestion
        {{- else if eq "development" .Chart.AppVersion }}
          image: gcr.io/kubecost1/cost-model-nightly:latest
```

---

Thanks very much. Fixed.